components/game/api/types.ts

```diff
@@ -1,11 +1,20 @@
-import { Field, GameId, Move, Player, PossibleCoords } from "../types";
+import {
+  Field,
+  GameId,
+  Move,
+  Player,
+  PlayerName,
+  PossibleCoords,
+} from "../types";
 
 export interface GameResponse {
   field: Field;
-  currentPlayer: Player;
+  currentPlayer: PlayerName;
```

Comment:
just a note, a bit cleaner from a data perspective would be to leave it 1/2 (or an enum) here